backend/src/main/resources/application.properties

```diff
@@ -15,3 +15,16 @@ spring.data.cassandra.cluster-name=Test Cluster
 spring.data.cassandra.local-datacenter=datacenter1
 spring.data.cassandra.keyspace-name=tokenkeyspace
 spring.data.cassandra.schema-action=CREATE_IF_NOT_EXISTS
+
+#   App Properties - Email Jwt Verification Token
+backend.app.jwtEmailVerificationSecret=emailVerificationSecretKey
+backend.app.jwtEmailVerificationExpirationMs=900000
+backend.app.confirmationLink=http://localhost:8080/users/confirm?token=
```

Comment:
Keep in mind that when you deploy this on server, the ip address (here localhost) will be the different